pages/docs/support/upgrading-grouparoo/v05-v06.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: "Upgrading Grouparoo from v0.5 to v0.6"
+date: "2021-08-31"
+pullQuote: "v0.6 is an exciting release with some breaking changes! Learn how to upgrade to v0.6."
+---
+
+## Calculated Properties
+
+v0.6 introduces calculated properties to Grouparoo. This allows you to run a custom javascript function to create properties that compute their value based on other properties.
+
+This is a new feature that does not introduce any breaking changes. Therefore, the only thing necessary to upgrade is to [follow the generic upgrade guide](/docs/support/upgrading-grouparoo#installing-updates) to bump your application to `v0.6.x`, and install the "Calculated Property" plugin.
+
+## Automatically deleting Profiles
+
+As of v0.6, Profiles are automatically deleted from Grouparoo when they've been removed from their primary source. This can be useful to make sure that profiles are removed from all of your destinations, which can be particularly helpful for GDPR compliance.
```

Comment:
```suggestion
As of v0.6, Profiles are automatically deleted from Grouparoo when they've been removed from their primary source. This can be useful to make sure that profiles are removed from all of your destinations, which can be particularly helpful for certain types of compliance.
```

I don't think I want to call out GDPR specifically